### ISSUE TITLE
Missing Release Functions?

### ISSUE THREAD

**Issue by GaryMcWhorter** [Trivaxy/WGPU.NET#8]
The webgpu.h header has release functions for most types, such as 
```c
WGPU_EXPORT void wgpuAdapterRelease(WGPUAdapter adapter) WGPU_FUNCTION_ATTRIBUTE;
```

I can't find any of these in the library. Is binding avoided?
I was expecting either `Wgpu.AdapterRelease(adapter)` or `adapter.Release()`

**Comment by Trivaxy** [comment]
They seem to have been included in the v0.16.1.1 update, while WGPU.NET currently targets v0.16.0.1.
It should only be a matter of updating the headers and API. I'll get to it as soon as I can.

If you need those functions _urgently_, the build instructions will help you generate the raw bindings using the new header files 👍

**Comment by GaryMcWhorter** [comment]
I didn't realize it was so recent 😅
No rush, I appreciate it. Thanks.
